VRCXDiscordTracker.Updater/Core/AppConstants.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+
+namespace VRCXDiscordTracker.Updater.Core;
+internal class AppConstants
+{
+    /// <summary>
+    /// アプリケーション名
+    /// </summary>
+    public static readonly string AppName = Assembly.GetExecutingAssembly().GetName().Name ?? string.Empty;
+
+    /// <summary>
+    /// アプリケーションバージョン
+    /// </summary>
+    public static readonly Version AppVersion = Assembly.GetExecutingAssembly().GetName().Version ?? new Version(0, 0, 0);
+
+    /// <summary>
+    /// アプリケーションバージョンの文字列
+    /// </summary>
+    public static readonly string AppVersionString = AppVersion.ToString(3); // Major.Minor.Patch
```

Comment:
現状だとバージョン情報はバージョン文字列以外に使う用途が無いし、`AppVersion`を消してひとまとめにしてしまうのも良いかもしれません。
もし必要であると考えるのなら、このままでも問題ありません。

```suggestion
    /// <summary>
    /// アプリケーションバージョンの文字列
    /// </summary>
    public static readonly string AppVersionString = (Assembly.GetExecutingAssembly().GetName().Version ?? new Version(0, 0, 0)).ToString(3); // Major.Minor.Patch
```